advanced/mathfx/src/main/java/org/arakhne/afc/math/geometry/d2/dfx/AbstractShape2dfx.java

```diff
@@ -54,23 +54,25 @@ public T clone() {
 		}
 	}
 
+	private boolean isEqualsToShape(Object obj) {
+		try {
+			return equalsToShape((T) obj);
+		} catch (ClassCastException exception) {
```

Comment:
This function has not the same behavior has its original code in `equals()`. If `(PathIterator2D<?>) obj` causes a `ClassCastException`, it will be no more catched.